configure.ac

```diff
@@ -62,12 +62,14 @@ AS_IF([test "x$using_demangle" != xyes],
 AC_ARG_ENABLE([strict], AS_HELP_STRING([--enable-strict], [Enable -Werror]), [], [enable_strict=yes])
 AM_CONDITIONAL([STRICT], [test "x$enable_strict" = xyes])
 
-AC_ARG_ENABLE([mpi],
-  AS_HELP_STRING([--disable-mpi], [Disable mpi sync daemon]), [found_mpi=yes], [found_mpi=no])
+# By default `--enable-mpi`, will set the "enable_mpi" variable
+AC_ARG_ENABLE([mpi], AS_HELP_STRING([--disable-mpi], [Disable mpi sync daemon]))
+
 AS_IF([test "x$enable_mpi" != xno], [
-  AX_MPI([found_mpi=yes], [found_mpi=no])
-])
-AM_CONDITIONAL([FOUND_MPI], [test "x$found_mpi" = xyes])
+  AX_MPI([enable_mpi=yes], [enable_mpi=no])
+  ])
+
+AM_CONDITIONAL([FOUND_MPI], [test "x$enable_mpi" != "xno"])
```

Comment:
https://stackoverflow.com/a/78146272 , soemthing related to that. Not sure to be honnest... But it seem to work for me.

aka `--disable-mpi` remove the `checking for mpicc... mpicc` on the log